src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -707,6 +724,29 @@ export class CellModel implements ICellModel {
 		return source;
 	}
 
+	// Run ADS commands via a notebook. Configured by a setting (turned off by default).
+	private checkForAdsCommandMagic(): boolean {
+		if (!this._isCommandExecutionSettingEnabled) {
+			return false;
+		}
+
+		let executeCommandMagic = '%%AZDATA_EXECUTE_COMMAND';
```

Comment:
Hmmmmm, not sure I like yet another use of azdata. It's not something I feel super strongly about - but what about just doing ADS_EXECUTE_COMMAND?

Reply:
Especially since a lot of guides will be mentioning use of the azdata tool I could see this becoming confusing for people.

Reply:
Yeah I was struggling with the name myself. Let me change that.